airbyte-integrations/connectors/destination-e2e-test/build.gradle

```diff
@@ -1,17 +1,36 @@
 plugins {
-    id 'airbyte-java-connector'
```

Comment:
this plugin is apparently where we get the `integrationTestJava` task from 🤦 adding it back locally for now, but we should figure out how to rewire our gradle stuff to get that working again